src/app/modules/main/profile_section/advanced/io_interface.nim

```diff
@@ -63,6 +63,9 @@ method toggleAutoMessage*(self: AccessInterface) {.base.} =
 method isDebugEnabled*(self: AccessInterface): bool {.base.} =
   raise newException(ValueError, "No implementation available")
 
+method isDebugSettingEnabled*(self: AccessInterface): bool {.base.} =
```

Comment:
I'd rather call this `runtimeLogLevelSet` (and invert the value).
Otherwise it's hard to guess the difference between `isDebugEnabled` and `isDebugSettingEnabled`.